docs/examples/graph-state.json

```diff
@@ -0,0 +1 @@
+{"focal_node":{"0":1,"1":1,"2":1,"3":1,"4":1,"5":1,"6":1,"7":1,"8":1,"9":1,"10":1,"11":1,"12":1,"13":1,"14":1,"15":1,"16":1,"17":1,"18":1,"19":1,"20":1,"21":2,"22":2,"23":2,"24":2,"25":2,"26":2,"27":2,"28":2,"29":2,"30":2,"31":2,"32":2,"33":2,"34":2,"35":2,"36":2,"37":2,"38":2,"39":2,"40":2,"41":2,"42":3,"43":3,"44":3,"45":3,"46":3,"47":3,"48":3,"49":3,"50":3,"51":3,"52":3,"53":3,"54":3,"55":3,"56":3,"57":3,"58":3,"59":3,"60":3,"61":3,"62":3},"theta":{"0":-3.1415926536,"1":-2.8274333882,"2":-2.5132741229,"3":-2.1991148575,"4":-1.8849555922,"5":-1.5707963268,"6":-1.2566370614,"7":-0.9424777961,"8":-0.6283185307,"9":-0.3141592654,"10":0.0,"11":0.3141592654,"12":0.6283185307,"13":0.9424777961,"14":1.2566370614,"15":1.5707963268,"16":1.8849555922,"17":2.1991148575,"18":2.5132741229,"19":2.8274333882,"20":3.1415926536,"21":-3.1415926536,"22":-2.8274333882,"23":-2.5132741229,"24":-2.1991148575,"25":-1.8849555922,"26":-1.5707963268,"27":-1.2566370614,"28":-0.9424777961,"29":-0.6283185307,"30":-0.3141592654,"31":0.0,"32":0.3141592654,"33":0.6283185307,"34":0.9424777961,"35":1.2566370614,"36":1.5707963268,"37":1.8849555922,"38":2.1991148575,"39":2.5132741229,"40":2.8274333882,"41":3.1415926536,"42":-3.1415926536,"43":-2.8274333882,"44":-2.5132741229,"45":-2.1991148575,"46":-1.8849555922,"47":-1.5707963268,"48":-1.2566370614,"49":-0.9424777961,"50":-0.6283185307,"51":-0.3141592654,"52":0.0,"53":0.3141592654,"54":0.6283185307,"55":0.9424777961,"56":1.2566370614,"57":1.5707963268,"58":1.8849555922,"59":2.1991148575,"60":2.5132741229,"61":2.8274333882,"62":3.1415926536},"n_bitstrings":{"0":1000,"1":1000,"2":1000,"3":1000,"4":1000,"5":1000,"6":1000,"7":1000,"8":1000,"9":1000,"10":1000,"11":1000,"12":1000,"13":1000,"14":1000,"15":1000,"16":1000,"17":1000,"18":1000,"19":1000,"20":1000,"21":1000,"22":1000,"23":1000,"24":1000,"25":1000,"26":1000,"27":1000,"28":1000,"29":1000,"30":1000,"31":1000,"32":1000,"33":1000,"34":1000,"35":1000,"36":1000,"37":1000,"38":1000,"39":1000,"40":1000,"41":1000,"42":1000,"43":1000,"44":1000,"45":1000,"46":1000,"47":1000,"48":1000,"49":1000,"50":1000,"51":1000,"52":1000,"53":1000,"54":1000,"55":1000,"56":1000,"57":1000,"58":1000,"59":1000,"60":1000,"61":1000,"62":1000},"avg_parity":{"0":0.531,"1":0.34,"2":0.212,"3":0.097,"4":0.032,"5":0.0,"6":0.015,"7":0.098,"8":0.217,"9":0.352,"10":0.523,"11":0.644,"12":0.795,"13":0.904,"14":0.976,"15":1.0,"16":0.973,"17":0.896,"18":0.809,"19":0.655,"20":0.48,"21":0.469,"22":0.314,"23":0.185,"24":0.097,"25":0.032,"26":0.0,"27":0.035,"28":0.106,"29":0.22,"30":0.354,"31":0.492,"32":0.658,"33":0.796,"34":0.893,"35":0.976,"36":1.0,"37":0.969,"38":0.889,"39":0.784,"40":0.679,"41":0.491,"42":0.507,"43":0.336,"44":0.203,"45":0.082,"46":0.036,"47":0.0,"48":0.024,"49":0.105,"50":0.204,"51":0.341,"52":0.479,"53":0.634,"54":0.78,"55":0.906,"56":0.969,"57":1.0,"58":0.973,"59":0.904,"60":0.797,"61":0.653,"62":0.491}}
```

Comment:
Not sure how this gets generated or if we want this, but it seems to pop up when running the tests

Reply:
OK, I'll gitignore it. Good call